src/Vector3.php

```diff
@@ -421,6 +429,14 @@ public static function max(Vector3 ...$positions) : Vector3{
 		return new Vector3(max($xList), max($yList), max($zList));
 	}
 
+	/**
+	 * Returns a new vector with the minimum xyz coordinates based from the passed vectors.
```

Comment:
It is more clear to say "Returns a new Vector3 taking the minimum of each component in the input vectors."
Another way to express this is "applies the PHP min function on each component over the input vectors".

Reply:
Perhaps minComponents() might be a better name with this in mind.